<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-QOO5UVKN" failures="1" tests="1" name="Test" time="25.659" errors="0" timestamp="2023-05-02T14:36:29 IST">
  <testcase classname="com.day9.Exercise5" name="test" time="7.292">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com/web/index.php/dashboard/index] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/web/index.php/dashboard/index] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at com.day9.Exercise5.test(Exercise5.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- test -->
</testsuite> <!-- Test -->
